src/js/index.js

```diff
@@ -21,15 +21,73 @@ const allowDifficultySelect = shouldAllow => sortingInput.querySelectorAll('.dif
 		node.disabled = !shouldAllow;
 	});
 
+const parameters = {
+	tags: {
+		default: '',
+		getValue: () => selectr.getValue().join(' '),
+		setValue: value => selectr.setValue(value.split(' '))
+	},
+	difficulty: {
+		default: 'all',
+		getValue: () => filterInput.value,
+		setValue: value => {
+			filterInput.value = ['easy', 'medium', 'hard'].includes(value) ?
+				value :
+				'all';
+		}
+	},
+	sort: {
+		default: 'alphabetical',
+		getValue: () => sortParams.sort.toLowerCase(),
+		setValue: value => {
+			sortParams.sort = ['alphabetical', 'difficulty'].includes(value) ?
+				value.toUpperCase() :
+				'ALPHABETICAL';
+		}
+	},
+	order: {
+		default: 'asc',
+		getValue: () => `${sortParams.order.split('SC')[0]}sc`.toLowerCase(),
+		setValue: value => {
+			sortParams.order = value === 'desc' ?
```

Comment:
Since these values are internal, we can just rename them

i.e. `option[value=alpha_asc]` rather than `option[value=ALPHABETICAL_ASCENDING]`